cli/commands/play.go

```diff
@@ -387,6 +399,7 @@ func getIndividualBoardStateForSnake(state *rules.BoardState, snake Battlesnake,
 		log.Panic("[PANIC]: Error Marshalling JSON from State")
 		panic(err)
 	}
+	fmt.Printf("%v\n", string(responseJson))
```

Comment:
I think we don't want the full request JSON logged every turn by default. Happy to add this with an optional parameter though.